src/backend/trace.rs

```diff
@@ -144,6 +144,15 @@ pub struct Hop {
     extensions: Option<Extensions>,
     mean: f64,
     m2: f64,
+    /// The ABS(RTTx - RTTx-n)
+    jitter: Option<Duration>,
+    /// The Sequential jitter average calculated for each
+    javg: f64,
```

Comment:
Given the logic in fn `javg_ms` to return this as an `Option`, perhaps it would make sense for `javg` to be an `Option`?

Reply:
I went ahead and made it an option.   I think this is an improvement and keeps the operations consistent for the Hop members.